responders/src/UI/embc-responder/src/app/feature-components/wizard/support-components/select-support/select-support.component.ts

```diff
@@ -31,7 +31,7 @@ export class SelectSupportComponent implements OnInit {
   ngOnInit(): void {
     this.supportList = this.loadEvacueeListService
       .getSupportTypeList()
-      .filter((element) => element.description !== '');
+      .filter((element) => element.description !== '' && element.value !== 'Lodging_Billeting');
```

Comment:
should this compare against the enum value and not text?

Reply:
it is pretty common that we compare against text  ( which is a value), so I would create a separate story to replace all the values with enum option values

Reply:
No story for this - just fix it as you see them please. Here is a good start.

Reply:
fixed